rust/worker/src/log/log.rs

```diff
@@ -80,6 +90,36 @@ pub(crate) struct GrpcLog {
 }
 
 impl GrpcLog {
+    pub(crate) fn try_wrap_request_with_trace_context<T>(
```

Comment:
Is this usable by all grpc clients in the rust code? should we move it out of this specific usage?